docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -402,17 +402,104 @@ feasible based on the data present, it is also a reasonable approach.
+[Cerdena_2020]_. Race-based research, which is characterized 
+by race essentialism, the notion that race is a biological factor (rather than a sociopolitical construct). 
+The practice of using race in this way has historically been the norm in US health research, and causes harm for 
+historically marginalized and minoritized groups, thereby exacerbating the very health inequities their health 
+research may be attempting to elucidate and alleviate. For examples of outdated race-based practices in medicine, 
+and their race-conscious alternatives, see Table 1 from Cerdena et al.’s 2020 paper [Cerdena_2020]_. 
+Race-conscious research, on the other hand, calls for a shift in focus from race to racism as a key determinant of 
+illness and health.
+
+Key recommendations from other sources
+--------------------------------------
+
+Here we have collected recommendations from various experts and guides advocating for new standards of communication 
+regarding racial health disparities.
+
+1. **Use a health equity lens when framing information about health disparities** [Calanan_2023]_.
+
+    Rather than allowing dominant narratives around race, individualism, and meritocracy limit our understanding of 
+    the root causes of health inequities, it is important to center a health equity perspective in health metrics 
+    research.  An equity lens allows us to recognize that systems of power and oppression (including white supremacy) 
+    shape institutional policies and living conditions that systematically harm populations 
+    [AMA_2021]_.
+
+2. **Use person-first language, avoid unintentional blaming, and be as specific as possible about the group to which you're referring**. 
+    
+    For more elaboration on these tips, please see Advancing Health Equity: A Guide to Language, Narrative and Concepts 
+    [AMA_2021]_.
+
+3. **Explicitly define race during experimental design, and specify the reason for its use in the study** [Boyd_2020]_. 
+    
+    As is described in our earlier section, `1.1 Research question`_, you should review all relevant 
+    social, environmental, and structural factors for which race may serve as a proxy measure when defining 
+    race as it pertains to your research,. "For the reader, these additional details enable careful interpretation 
+    of study results and implications. But for authors, it engenders critical thinking about racial constructs that 
+    prevent the reification of race as a biological entity." [Boyd_2020]_ 
+
+    Health metrics researchers should embrace a critical race theory (CRT) framework, which provides "tools for conducting research and practice 
+    are intended to elucidate contemporary racial phenomena, expand the vocabulary with which to discuss complex racial concepts, and 
+    challenge racial hierarchies" [Ford_Airhihenbuwa_2010]_. Embracing a CRT framework might entail
+    some of the following: 
```

Comment:
I feel like this might be missing a sentence. I kept expecting you to define CRT here. I think maybe a sentence that said CRT's core belief is XYZ and based on this, it provides LIST. Embracing this in practice might entail BULLETS.

Reply:
There doesn't seem to be a succinct definition in the article, but the article lists "4 of its basic features: race consciousness, contemporary orientation, centering in the margins rather than in the mainstream, and praxis (i.e., theory-informed action)."

The Origins section says:
> Critical Race Theory is not like behavior change or epidemiological theories. Rather, it is an iterative methodology for helping investigators remain attentive to equity while carrying out research, scholarship, and practice. ... Knowledge production is the primary medium through which Critical Race Theory operates. The scholarship distinguishes contemporary racial mechanisms from older ones (e.g., Jim Crowism), expands the vocabulary for discussing racial phenomena and investigating racism effects, and explicitly incorporates the knowledge of racial and ethnic minority communities regarding marginality.

Not sure if any of that gets at what @SylLutze is looking for.

Reply:
@pletale You were wondering in coworking whether the CRT section was too long. It does seem a bit long compared to the other recommendations. Here are some thoughts:

I think it should probably be its own numbered recommendation, rather than being lumped in with the "Explicitly define race during experimental design" recommendation. And I think it should probably be the _last_ numbered recommendation, or perhaps a separate paragraph after the list of recommendations, as it's describing a more comprehensive methodology while the others are more discrete things you could cross off a checklist as you're doing your research.

If you wanted to trim it down, you could probably provide a bit less detail, and do it more like the other recommendations where you list some of the basic elements, but refer the reader to the citation if they want to know more. But I also think it's fine to provide more detail if you want to leave it in, since CRT does seem to involve a lot of elements, and if we want to apply it, we need to understand what they are.